frontend/src/components/Topics/Topic/Messages/Filters/Filters.tsx

```diff
@@ -187,7 +284,84 @@ const Filters: React.FC<FiltersProps> = ({
           </Button>
         </FlexBox>
 
-        <Search placeholder="Search" value={search} onChange={setSearch} />
+        <FlexBox gap="8px" alignItems="center">
+          <Search placeholder="Search" value={search} onChange={setSearch} />
+          <div style={{ position: 'relative' }}>
+            <Button
+              disabled={isFetching || messages.length === 0}
+              buttonType="secondary"
+              buttonSize="M"
+              onClick={handleDownloadClick}
+              style={{
+                minWidth: '40px',
+                padding: '8px',
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+              }}
+            >
+              <svg
+                width="24"
+                height="24"
+                viewBox="0 0 18 18"
+                fill="currentColor"
+              >
+                <path d="M4.24 5.8a.75.75 0 001.06-.04l1.95-2.1v6.59a.75.75 0 001.5 0V3.66l1.95 2.1a.75.75 0 101.1-1.02l-3.25-3.5a.75.75 0 00-1.101.001L4.2 4.74a.75.75 0 00.04 1.06z" />
```

Comment:
Can we please extract this icon like others into a separate file/class in `src/components/common/Icons`?